src/Dropdown.vue

```diff
@@ -46,12 +46,16 @@ export default {
     },
     type: {
       type: String,
-      default: 'default'
+      default: 'light'
     }
   },
   computed: {
     btnType () {
-      return `btn-${this.type}`;
+      if (this.type !== '') {
+        return `btn-${this.type}`;
+      } else {
+        return '';
```

Comment:
Empty String won't appear in the logic as the default value is 'light', unless the user explicitly passes in an empty string (unlikely). You should refactor to account for this.